pkg/auth/oauth/registry/tokentimeouts.go

```diff
@@ -133,27 +142,31 @@ func (a *oauthTokenTimeoutValidator) updateTokenSeen(td tokenData) {
 }
 
 func (a *oauthTokenTimeoutValidator) updateTimeouts(clientTimeout int32) {
-	timeout := int32(math.Ceil(float64(clientTimeout) / 3.0))
+	// timeout is set to one third of clientTimeout
+	timeout := clientTimeout / 3
```

Comment:
Ceil made this be 1/3 or larger, rather then possibly smaller.
I guess it doesn't matter a whole lot given we decided the min timeout is 2m30s ...

Reply:
Yeah it seemed to be too complex for too little benefit.